neqo-crypto/src/time.rs

```diff
@@ -180,7 +180,7 @@ mod test {
 
     #[test]
     fn convert_stable() {
-        let now = Time::from(Instant::now());
+        let now = Time::now();
```

Comment:
this looks like an unrelated change that may be the source of the CI trouble, so just omit from this PR please.